src/ArrayCache.php

```diff
@@ -1,49 +1,175 @@
 <?php
+
 namespace Yiisoft\Cache;
 
-/**
- * ArrayCache provides caching for the current request only by storing the values in an array.
- *
- * See {@see \Psr\SimpleCache\CacheInterface} for common cache operations that ArrayCache supports.
- */
-final class ArrayCache extends SimpleCache
+final class ArrayCache extends BaseCache
 {
-    private const TTL_INFINITY = 0;
+    private $cache = [];
 
     /**
-     * @var array cached values.
+     * Fetches a value from the cache.
```

Comment:
There's no need to duplicate interface phpdoc.

Reply:
Agree